rmgpy/kinetics/surface.pyx

```diff
@@ -406,6 +478,21 @@ cdef class SurfaceArrhenius(Arrhenius):
         def __set__(self, value):
             self._A = quantity.SurfaceRateCoefficient(value)
 
+    property cov:
```

Comment:
I like coverage. I can change them all to coverage.

Reply:
Hmmm in `solver/surface.pyx` there is already a `coverage` that has to do with current surface coverage so I will change the name to `coverage_dependence` to reduce confusion. I also realized there is a covariance `cov` as well that could have been getting caught up in the shuffle

Reply:
@rwest do you think it should stay `cov` when we enter it in as part of the data in the training reactions, or should that be expanded to `coverage_dependence` as well? I chose cov because that is what chemkin uses

Reply:
I wouldn't stick to what CHEMKIN did in the 1960's. 
[Cantera YAML](https://cantera.org/documentation/docs-2.5/sphinx/html/yaml/reactions.html#interface) now uses `coverage-dependencies: {H(s): {a: 0, m: 0, E: -6000 J/mol}}`  for example.
If we're going with `coverage_dependence` as the parameter name internally, I think use the same in the input file.

Reply:
Or copy Cantera and call it dependencies not dependence